packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -231,6 +239,7 @@ class MiniCart extends Component {
                     </Trigger>
                 </div>
                 {miniCartInner}
+                <Mask isActive={isMiniCartMaskOpen} dismiss={cancelCheckout} />
```

Comment:
I know I'm late to the party here but do we need a component for this? Could it just be a div with a `className` and an `onClick` rendered by a conditional? ie
```js
{ isOpen ? <div className={classes.mask} onClick={cancelCheckout}></div> : null }
```